modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_3_1/UpgradeDDMFormFieldValidation.java

```diff
@@ -227,35 +224,35 @@ private void _upgradeFields(
 
 					if (expressionJSONObject == null) {
 						_upgradeValidation(
-							availableLanguageIdsJSONArray, validationJSONObject,
-							expressionString);
+							availableLanguageIdsJSONArray, expression,
+							validationJSONObject);
 					}
 					else if (Validator.isNull(
-								expressionJSONObject.getString("name"))) {
+								expressionJSONObject.isNull("name"))) {
```

Comment:
This will change the execution. I assume you intended to remove the `Validator.isNull` as well, right @leoadb?

Reply:
@rodrigopaulino yes! can you please send me a follow up PR? Thanks!

Reply:
https://github.com/leoadb/liferay-portal/pull/627